libkbfs/block_journal.go

```diff
@@ -634,6 +634,8 @@ func (j *blockJournal) flushOne(
 			return false, err
 		}
 
+		// TODO: If the reference add fails, retry with a
+		// Put. This is tricky: see KBFS-1148 and KBFS-1255.
```

Comment:
Maybe if `isRecoverableBlockError(err)`, you can print out a warning message here so we can easily notice it when it happens during our testing?  Though I guess if you completely turn off deduping we won't need that.